### ISSUE TITLE
Annoying warning messages in OSX 10.11 (El Capitan)

### ISSUE THREAD

**Issue by ataiya** [textmate/latex.tmbundle#148]
I've upgraded to "El Capitan" and something just became quite worse. I have a document that compiles just fine, but regularly I get the error window **popping** up and sticking around for a second or two, then it **disappears** by itself. If I compile the latex document with Command+R there is no problem... this only happens in the "watch document" mode. Furthermore, I cannot even check the log file, as after the message disappears I think the problem is self-resolved (so the log file doesn't contain any error at all)...

This is a snapshot of the problem.
Anybody having the same issue?
(this happens on two different machines, and only after I updated the OS)

![screenshot 2016-04-16 21 36 18](https://cloud.githubusercontent.com/assets/4480573/14585088/ce727466-041b-11e6-824f-78a3c4eaaa57.png)

**Comment by sanssecours** [comment]
Hi Andrea,

thanks for the bug report. This issue should have nothing to do with the operating system you are using. The pop-up shows that `latexmk` returned a non-zero return code. It looks like the document contains some form of (minimal) error. If you are sure that the document contains no errors, then you can also try to remove all auxiliary files (`^` + `⌥` + `⌫`), before you use “Watch document”. Then the pop-up should not show up again, until you save a faulty version of the document.

Can you please send me a (minimal) example document and steps to reproduce the error?

Thank you,
  René

**Comment by ataiya** [comment]
Hi Rene, unfortunately I do not have a minimal example. I am working on a deadline and the working repository is all I have. If you give me an email I can send you a zip (to be kept confidential) and you can try re-compile the project yourself?

**Comment by sanssecours** [comment]
> If you give me an email I can send you a zip (to be kept confidential) and you can try re-compile the project yourself?

Okay. Please send the files to sanssecours@me.com.

**Comment by ataiya** [comment]
Apparently this was my mistake. There was an error inside my .bib file. Can I change my bug report? I'd say that this plugin need to find a better way of intercepting and showing bibtex errors!!!

**Comment by sanssecours** [comment]
> Can I change my bug report? I'd say that this plugin need to find a better way of intercepting and showing bibtex errors!!!

I opened a new issue: #149

**Comment by ataiya** [comment]
Awesome, thanks :)​
